.github/workflows/swift.yaml

```diff
@@ -0,0 +1,106 @@
+name: Swift CI
+
+on:
+  push:
+    branches:
+      - master
+      - '!gh-pages'
+    paths:
+      - '*.podspec'
+      - 'Gemfile*'
+      - 'Package.swift'
+      - 'swift/**'
+      - '.github/workflows/swift.yaml'
+  pull_request:
+    paths:
+      - '*.podspec'
+      - 'Gemfile*'
+      - 'Package.swift'
+      - 'swift/**'
+      - '.github/workflows/swift.yaml'
+
+jobs:
+  development-apps:
+    runs-on: macOS-latest
+
+    strategy:
+      matrix:
+        scheme: 
+        - Development-Unit-WorkflowTests
+        - Development-Unit-WorkflowUITests
+        - Development-Unit-SplitScreenTests
+        - Development-Unit-TicTacToeTests
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install Bundler
+      run: |
+        gem update --system
+        gem install bundler
+
+    - name: Bundle Install
+      run: bundle install
+
+    - name: Pod Install
+      run: |
+        bundle exec pod gen Development.podspec
+
+    - name: Switch Xcode
+      run: sudo xcode-select -s /Applications/Xcode_11.3.1.app
+
+    - name: Build & Test
+      run: |
+        set -o pipefail && xcodebuild -workspace gen/Development/Development.xcworkspace -scheme ${{ matrix.scheme }} -destination platform\=iOS\ Simulator,OS\=13.3,name\=iPad\ Pro\ \(9.7-inch\) build test | xcpretty
+
+  spm:
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Swift Package Manager - iOS
+      run: |
+        xcodebuild -scheme "Workflow-Package" test -destination "name=iPhone 11"
+
+    - name: Swift Package Manager - macOS
+      run: |
+        xcodebuild -scheme "Workflow-Package" test
+
+  tutorial:
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install Bundler
+      run: |
+        gem update --system
+        gem install bundler
+
+    - name: Bundle Install
+      run: bundle install
+
+    - name: Tutorial App
+      run: |
+        cd swift/Samples/Tutorial
+        bundle exec pod repo update
+        bundle exec pod install
+        set -o pipefail && xcodebuild -workspace Tutorial.xcworkspace -scheme Tutorial -destination platform\=iOS\ Simulator,OS\=13.3,name\=iPad\ Pro\ \(9.7-inch\) build test | xcpretty
+
+  documentation-lint:
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install
+      run: |
+        gem update --system
+        gem install bundler
+        bundle install
+        brew install sourcedocs
+        
+    - name: Swiftdocs
+      run: |
+        .buildscript/build_swift_docs.sh /tmp/swiftdocs
```

Comment:
Super nit (don't block the PR on this): `${{ runner.temp }}` [is a thing](https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#runner-context). Seems like the only benefit of using it is that it's guaranteed to be empty, which doesn't matter here.